scripts/ci/analyze-commits.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
```

Comment:
I wonder if this would be faster with a bash script that directly greps the output of the `git diff`.

@hansl what do you think?